src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -128,25 +128,25 @@
 		[big]The protocol[/big][br]
 		Well, there is no protocol actually, only the existing implementations, that
 		can be accessed by anyone that want to reproduce them. There are only some
-		points relating to the crypting engines that need to be cleared:[br]
-		The crypted text must be suitable to be sent through an IRC connection;
-		this means that some characters can not appear in the crypted text (e.g. CR,LF,NULL....).
-		KVIrc solves it in a simple way: the crypted binary data is encoded,
+		points relating to the encrypting engines that need to be cleared:[br]
+		The encrypted text must be suitable to be sent through an IRC connection;
+		this means that some characters can not appear in the encrypted text (e.g. CR,LF,NULL....).
+		KVIrc solves it in a simple way: the encrypted binary data is encoded,
 		either as hexadecimal numeric string or in base64.[br]
-		An escape character has been defined to identify messages that are "crypted" from the
-		ones that are not: this character has ascii code 30 (decimal).[br]
+		An escape character has been defined to identify messages that are "encrypted" from the
+		ones that are not: this character has ASCII code 30 (decimal).[br]
 		The encoding is used in private messages only and has the following format:[br]
 		PRIVMSG <target> :<escape_char_ascii_30><encrypted message>[br]
 		ASCII 30 does not correspond to any widely used escape sequence and allows
 		mixing encrypted and plain text messages in a conversation...Well, this is not
-		so pretty but you can exchange crypted messages with one or two friends while
+		so pretty but you can exchange encrypted messages with one or two friends while
 		talking on a normal IRC channel: nobody else than your friends will be able
```

Comment:
I don't like that there are `:`s and `;`s used here. It'd be better to just replace them with periods, otherwise the sentence structures just become too complex. We know all 3 statements are intertwined with each other so no need to over do it.